enterprise/server/remote_execution/containers/docker/docker.go

```diff
@@ -135,21 +137,32 @@ func (r *dockerCommandContainer) Run(ctx context.Context, command *repb.Command,
 		result.Error = wrapDockerErr(err, "failed to create docker container")
 		return result
 	}
+	cid := createResp.ID
+
```

Comment:
we can use `containerName` instead of `cid` for the attach request to simplify a tiny bit.

Reply:
I will stick with cid to show that the Attach call depends on ContainerCreate call and consistent with the other docker client API calls.